internal/utils/fshelper.go

```diff
@@ -0,0 +1,47 @@
+package utils
```

Comment:
Can we have a full name for this file? `filesystemHelper.go`?

Reply:
changed the name. Regarding its own package: we currently have a lot of minor interfaces/functionalities in utils. We can discuss if we want to dismantle this package in a later PRs

Reply:
OK then.